src/components/ChooseYourCocktail.jsx

```diff
@@ -11,15 +12,23 @@ function ChooseYourCocktail({ cocktailList }) {
   const query = useQuery();
   const page = parseInt((query.get('page') ?? 1), 10);
   const pageSize = 3;
+  const cocktailListLength = cocktailList.length / 3;
```

Comment:
that may not lead to an exact value, isn't it ?

if I have 17 coktails, and a page size of 3, your cocktailsListLength will allow 17/3 = 5 page. I will miss the 6th page with the last 2 cocktails ;)